### DISCUSSION TITLE
Unexplained TypeScript error.

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [aralroca/fragstore#69] [Q&A]
I know you do not have full typescript support yet but version 9 claims to have some.
Since I am using React (16.13.1) in SPFx  I have to use typescript.
Even for the minimal setup in store.ts
```
import createStore from "teaful";
export const { useStore } = createStore();
```
I get an error on "useStore" :
`Exported variable 'useStore' has or is using name 'useStoreType' from external module "teaful" but cannot be named.ts(4023)`

Can anyone help with some boilerplate usage with TypeScript? 
useStore must be exported to be used in other components.

Thank you

**Comment by ruslan-byondxr** [comment]
@Ofer-Gal Please share a minimal reproduction with us

**Reply by Ofer-Gal** [reply]
This was it!
Only 2 lines in store.ts Taken from your README.md. 
If I use the example "How to export" 
```
// ✅
export const { useStore, getStore, withStore } = createStore({
  cart: { price: 0, items: [] },
});
```
I get the same error for useStore, getStore and withStore
Maybe we are not on same typescript version? mine is 3.9

Thanks
